internal/scaffold/main.go

```diff
@@ -55,12 +55,12 @@ func main() {
 
 	app, err := service.New(service.Parameters{ApplicationStartInfo: info, Factories: factories})
 	if err != nil {
-		log.Fatal("failed to construct the application: %w", err)
+		log.Fatalf("failed to construct the application: %v", err)
```

Comment:
Are those the only places affected?

Reply:
In this repo it does seem like it. The only other places using `%w` are `fmt.Errorf()` calls which does support it.